File: C/EX35.c
#include<stdio.h>
#include<stdlib.h>

int main(int argc, char const *argv[])
{
    //declaração da variavel e do vetor
    int num;
    int vetor[20];

    printf("Digite 20 numeros.\n");

    //preenchimento do vetor
    for (int i = 0; i < 20; i++)
    {
        printf("n: ");
        scanf("%i", &vetor[i]);
    }
    
    printf("Digite o fator multiplicativo: ");
    scanf("%i", &num);

    //multiplivação de 'num' por cada um dos elementos do vetor
    for (int i = 0; i < 20; i++)
    {
        vetor[i] = num * vetor[i];
    }

    //apresentação do vetor com os resultados das multiplicações
    for (int i = 0; i < 20; i++)
    {
        printf("%3i", vetor[i]);
    }
    
    

    return 0;
}

File: Python/EX35.py
numeros=[]

for i in range(0, 20, 1):
    x=int(input("N%d: " %(i+1)))
    numeros.append(x)

n=int(input("Fator multiplicativo: "))

for i in range(0, 20, 1):
    numeros[i] *= n

print(f"vetor: {numeros}")





'''
numeros = list(range(20))
x=0

print("Digite os valores.")

#preenchimento do vetor - a função len() retorna o tamanho do vetor
while x < len(numeros):
    numeros[x] = int(input("N%d: " % x))
    x+=1

#imprime o vetor digitado
print(f"Vetor: {numeros}")

#solicita o valor para multiplicação
num = int(input("Fator de multiplcação: "))

#reescreve o vetor com o resultado dos produtos
for e in range(0,20,1):
    numeros[e] *= num 

#reimprime vetor
print(f"Vetor: {numeros}")
'''